composer.json

```diff
@@ -11,12 +11,13 @@
     }
   ],
   "require": {
-    "php": "^7.0",
+    "php": "^7.1",
     "geocoder-php/plugin": "^1.0",
     "symfony/console": "^3.4 || ^4.2",
     "symfony/framework-bundle": "^3.4 || ^4.2",
     "symfony/options-resolver": "^3.4 || ^4.2",
-    "willdurand/geocoder": "^4.0"
+    "willdurand/geocoder": "^4.0",
+    "php-http/discovery": "^1.7"
```

Comment:
Did you add this manually?

Reply:
No, I didn't... or maybe. :/

Reply:
Strange, I was sure that I've added sorting config to composer.json in another PR 🤦‍♂

Reply:
I saw that you reordered them. But I guess you missed adding the config.. Anyhow I did that now